pallets/dactr/src/weights.rs

```diff
@@ -145,10 +153,17 @@ impl WeightInfo for () {
 	}
 	/// The range of component `i` is `[0, 524288]`.
 	fn data_root(i: u32, ) -> Weight {
-		// Minimum execution time: 609 nanoseconds.
-		Weight::from_ref_time(672_000_u64)
-			// Standard Error: 12
-			.saturating_add(Weight::from_ref_time(4_321_u64).saturating_mul(i as u64))
+		// Minimum execution time: 1_457 nanoseconds.
+		Weight::from_ref_time(1_487_000_u64)
+			// Standard Error: 21
+			.saturating_add(Weight::from_ref_time(5_251_u64).saturating_mul(i as u64))
+	}
+	/// The range of component `i` is `[0, 2097152]`.
+	fn data_root_batch(i: u32, ) -> Weight {
+		// Minimum execution time: 778 nanoseconds.
+		Weight::from_ref_time(825_000_u64)
```

Comment:
It is weird that one `data_root` takes `1_487` as minimum but several `data_root` takes `824` (less). 
I mean It should be equal or greater, right?

Reply:
I guess when the range is greater the results are smoother.